doc/modules/neural_networks_supervised.rst

```diff
@@ -7,6 +7,12 @@ Neural network models (supervised)
 .. currentmodule:: sklearn.neural_network
 
 
+.. warning::
+
+    The MLP implementation here should not be used for any serious neural network or
```

Comment:
I don't like the expression "serious neural network" (I know it came from @GaelVaroquaux). Maybe say "This implementation is not intended for large-scale applications. In particular, scikit-learn offers no GPU support. For much faster, GPU-based implementations, as well as frameworks offering much more flexibility to build deep learning architectures, see X". This might be too long, though. @kastnerkyle knows the ideal way to phrase this, I'm sure ;)